### PR TITLE
Update examples  to use ES6 classes

### PR DESCRIPTION
**Pull request by svenkatreddy**
Examples update to use ES6 classes

Update to constructor methods, default Properties and bindings.

### CODE REVIEW

**Comment by svenkatreddy** [comment]
@zpao can you check this one too ?

File: examples/dynamic/views/Html.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ var React = require('react');
 var ReactDOMServer = require('react-dom/server');
 var Content = require('./Content');
 
-module.exports = React.createClass({
+export default class Html extends React.Component {
```
Comment (zpao) [inline examples/dynamic/views/Html.js]: Please leave this as `module.exports` as we're still using commonjs modules for the rest of this. You may need to create the class first before assigning to `module.exports`.

File: examples/dynamic/views/Content.js [hunk 2]
```diff
@@ -38,6 +44,6 @@ var TodoApp = React.createClass({
       </div>
     );
   }
-});
+}
 
-module.exports = TodoApp;
+module.exports = TodoApp;
```
Comment (zpao) [inline examples/dynamic/views/Content.js]: Please leave the trailing newline. You shouldn't have to touch this line at all.

File: examples/dynamic/views/Content.js [hunk 3]
```diff
@@ -1,29 +1,35 @@
 var React = require('react');
 
-var TodoList = React.createClass({
-  render: function() {
+class TodoList extends React.Component {
+  constructor(props) {
```
Comment (zpao) [inline examples/dynamic/views/Content.js]: Let's leave the constructor out entirely. It's not needed if all you call is `super()`.

**Comment by svenkatreddy** [comment]
@zpao Updated!

**Comment by zpao** [comment]
👍 thanks!

**Pull request merged**